docs/authoring.md

```diff
@@ -183,6 +183,7 @@ The `.tp-config.json` file contains a JSON object that may have the following in
 |`supportedProtocols`| [String] | A list with all the protocols this template supports.
 |`parameters`| Object[String, Object] | An object with all the parameters that can be passed when generating the template. When using the command line, it's done by indicating `--param name=value` or `-p name=value`.
 |`parameters[param].description`| String | A user-friendly description about the parameter.
+|`parameters[param].default`| String | Default value of this parameter if not set in the start command.
```

Comment:
What is "start command"? Also, from CLI everything may be String but when using generator as a module it can be anything. I'd put Any as the type.

Reply:
I agree that type is `Any`, didn't know how to properly name it, thanks!
"Start command" is `ag asyncapi.yaml @asyncapi/html-template`, how to name it to be more clear?

Reply:
the thing is that as @fmvilas mentioned, it is not only used in CLI but generator can be used as a library too. Just call it `Default value of the parameter if not specified.` maybe?

Reply:
Agree, used.